doc/source/ray-observability/user-guides/index.md

```diff
@@ -5,8 +5,6 @@
 These guides help you monitor and debug your Ray applications and clusters.
 
 The guides include:
-* {ref}`observability-general-troubleshoot`
-* {ref}`observability-debug-apps`
```

Comment:
Why do we remove both of them? We can still keep the 2nd one?

Reply:
Thanks for catching. Will keep the 2nd one!